docs/user/connectors/action-types.asciidoc

```diff
@@ -1,11 +1,11 @@
 [role="xpack"]
```

Comment:
This page doesn't seem to render properly.

<img width="499" alt="Screen Shot 2021-04-05 at 9 19 37 AM" src="https://user-images.githubusercontent.com/3694571/113577969-1c096700-95f0-11eb-9690-9064a0d6f828.png">